### DISCUSSION TITLE
02-微信支付

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#28] [备发布站的评论系统]
# 02-微信支付

02-微信支付N100充电桩，框架为springboot，需要在app模块，也就是小程序的后端服务中，使用接微信支付；在网上选择了很多的技术选型，发现如果是自己封装，一个是太繁琐，还有就是不方便维护，发现一个比较好的springboot包官网地址：Document (notfound403.github.io)S...

http://127.0.0.1:4000/02-%E5%BE%AE%E4%BF%A1%E6%94%AF%E4%BB%98

**Comment by zpc2222** [comment]
牛pi
